frontend/src/components/AcmSelectBase.tsx

```diff
@@ -182,11 +182,14 @@ export function AcmSelectBase(props: AcmSelectBaseProps) {
   let placeholder = props.placeholderText
   if (!placeholder) {
     if (isMulti && selections.length > 0) {
-      placeholder = selections.join(', ')
+      placeholder =
+        variant !== SelectVariant.typeaheadMulti ? selections.join(', ') : props.placeholder ?? t('Select choices')
```

Comment:
2 questions:
- why do we have both placeholderText and placeholder props?
- why do we use `t('Select choices')` here and `''` on line 197 when `props.placeholder` is nullish? (I would prefer not to introduce a default string here and just supply the relevant placeholder if you noticed we were missing one somewhere... we could make the prop required)